authentication.go

```diff
@@ -88,12 +88,12 @@ type UserAccessTokenResponse struct {
 }
 
 type accessTokenRequestData struct {
-	Code         string   `query:"code"`
-	ClientID     string   `query:"client_id"`
-	ClientSecret string   `query:"client_secret"`
-	RedirectURI  string   `query:"redirect_uri"`
-	GrantType    string   `query:"grant_type"`
-	Scopes       []string `query:"scope"`
+	Code         string `query:"code"`
+	ClientID     string `query:"client_id"`
+	ClientSecret string `query:"client_secret"`
+	RedirectURI  string `query:"redirect_uri"`
+	GrantType    string `query:"grant_type"`
+	Scopes       string `query:"scope"`
```

Comment:
`Scopes` throughout this project are a slice of strings. So let's revert this change and instead action the following comment.